ui/plugins/bookmarks/src/components/entry-renderer.tsx

```diff
@@ -2,14 +2,15 @@ import React from 'react';
 import DS from '@akashaproject/design-system';
 import { useTranslation } from 'react-i18next';
 import { useIsFollowing } from '@akashaproject/ui-awf-hooks/lib/use-follow.new';
-import { ItemTypes } from '@akashaproject/ui-awf-typings/lib/app-loader';
+import { EventTypes, ItemTypes } from '@akashaproject/ui-awf-typings/lib/app-loader';
 import { IEntryData } from '@akashaproject/ui-awf-typings/lib/entry';
 import { usePost } from '@akashaproject/ui-awf-hooks/lib/use-posts.new';
 import { useComment } from '@akashaproject/ui-awf-hooks/lib/use-comments.new';
 import { mapEntry } from '@akashaproject/ui-awf-hooks/lib/utils/entry-utils';
 import { ILogger } from '@akashaproject/sdk-typings/lib/interfaces/log';
 import { RootComponentProps } from '@akashaproject/ui-awf-typings';
 import { ILocale } from '@akashaproject/design-system/lib/utils/time';
+import ExtensionPoint from '@akashaproject/design-system/lib/utils/extension-point';
```

Comment:
Codacy has a fix for the issue: [Replace `ExtensionPoint·from·'@akashaproject/design-system/lib/utils/extension-poin` with `React·from·'react';⏎import·{·useTranslation·}·from·'react-i18nex`](https://app.codacy.com/gh/AKASHAorg/akasha-world-framework/pullRequest?prid=8098750)
```suggestion
import React from 'react';
```

Reply:
ignore this, it's a tool that I'm testing

Reply:
it's dumb af